api/swagger-spec/resourceListing.json

```diff
@@ -10,6 +10,14 @@
     "description": "get available API versions"
    },
    {
+    "path": "/apis/experimental/v1alpha1",
```

Comment:
We need spec files for these new paths.
You need to add these paths here: https://github.com/kubernetes/kubernetes/blob/master/hack/after-build/update-swagger-spec.sh#L72

PS: I see now that someone added /apis without adding the spec for it. Please fix that as well, while you are at it.